sdks/java/core/src/main/java/org/apache/beam/sdk/testing/CoderProperties.java

```diff
@@ -104,6 +104,16 @@
     assertThat(decodeEncode(coder, context, value), equalTo(value));
   }
 
+  /**
+   * Verifies that for the given {@code Coder<T>}, {@code Coder.Context}, and value of type {@code
+   * T}, encoding followed by decoding yields an equal value of type {@code T}.
```

Comment:
This one doesn't test that it is equal, but tests that the matcher succeeds.

Reply:
Changed description